src/types/onyx/Task.ts

```diff
@@ -25,6 +25,9 @@ type Task = {
 
     /** Chat report with assignee of task */
     assigneeChatReport?: Report;
+
+    /** If set, skip confirmation when creating the task */
+    skipConfirmation?: boolean;
```

Comment:
Why is this optional here but not in transaction?

Reply:
TransactionPropTypes is still in js 😅

Reply:
Ahh that makes sense then